cognite/neat/_rules/models/dms/_validation.py

```diff
@@ -385,8 +410,8 @@ def _validate_reverse_connections(
         for (view_id, prop_id), prop_ in properties_by_ids.items():
             if not isinstance(prop_, ReverseDirectRelationApply | ReverseDirectRelation):
                 continue
-            source_id = prop_.through.source, prop_.through.property
-            if source_id not in properties_by_ids:
+            target_id = prop_.through.source, prop_.through.property
+            if target_id not in properties_by_ids:
```

Comment:
More descriptive name. Target of reverse direct relation, instead of source of reverse direct relation. Up for discussion